paper.tex

```diff
@@ -845,14 +847,19 @@ \subsection*{ARG likelihood} \label{par:liks}
 % intervals; rate changes along the genome can be incorporated similarly.
 Note that we have only considered the topology and branch length information here
 an not considered mutational information.
-The likelihood of a set of mutations given an ARG is
+The likelihood of a set of mutations given an ARG $\mathcal{G}$ given mutation rate $\mu$ is
 \begin{equation}\label{eq:lik-mut}
     \mathcal{L}(\mathcal{D}|\mathcal{G}, \mu)
     =
-    \frac{1}{\mathcal{M}!} \prod_{(p, c, x, y) \in E} e^{-(y-x)(t_p-t_c)\mu^{|m_e|}}
+    \prod_{e \in E} \frac{(\mu(t_p-t_c)(y-x))^{m_e}}{m_e!} e^{-\mu (t_p-t_c)(y-x)} ,
+    % I don't think the following is correct:
+    % \frac{1}{\mathcal{M}!} \prod_{(p, c, x, y) \in E} e^{-(y-x)(t_p-t_c)\mu^{|m_e|}}
```

Comment:
maybe I'm missing something? The previous expression did not look right to me at all; I've replaced it with the Poisson expression.

Reply:
We could just leave it out and cite some papers, there's nothing new here.